rmw_implementation/package.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0"?>
-<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
-<package format="2">
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
```

Comment:
I think right now nothing in the ROS eco-system (1 or 2) requires it. Actually tinyxml(2) just ignores any `<?...?>` tag.
More advanced parsers can use it and complain if the following xml string doesn't validate the provided schema.

Reply:
If you have your editor configured to lint xml files (e.g. in Atom using `linter-xmllint`) it will use the referenced schema. Since the schema is not yet final it isn't available through that URL but you can reference the file with a local `file://` to point to the current version in the branch on the REP repo.